src/Factory/ContentStructureFactory.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeRhapsodie\EzDataflowBundle\Factory;
+
+use CodeRhapsodie\EzDataflowBundle\Model\ContentCreateStructure;
+use CodeRhapsodie\EzDataflowBundle\Model\ContentStructure;
+use CodeRhapsodie\EzDataflowBundle\Model\ContentUpdateStructure;
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+
+class ContentStructureFactory
+{
+    /**
+     * @var ContentService
+     */
+    private $contentService;
+
+    /**
+     * ContentStructureFactory constructor.
+     *
+     * @param ContentService $contentService
+     */
+    public function __construct(ContentService $contentService)
+    {
+        $this->contentService = $contentService;
+    }
+
+    /**
+     * @param array     $data
+     * @param string    $remoteId
+     * @param string    $language
+     * @param string    $contentType
+     * @param int|int[] $parentLocationId
```

Comment:
It can be several types so let's just put `mixed` and rename the argument.
```suggestion
     * @param mixed $parentLocations
```